test/framework/src/main/java/org/opensearch/test/ClusterServiceUtils.java

```diff
@@ -169,8 +171,18 @@ public static ClusterService createClusterService(ThreadPool threadPool, Discove
     }
 
     public static ClusterService createClusterService(ThreadPool threadPool, DiscoveryNode localNode, ClusterSettings clusterSettings) {
+        MetricsRegistry metricsRegistry = new NoopMetricsRegistryFactory().getMetricsRegistry();
+        return createClusterService(threadPool, localNode, clusterSettings, metricsRegistry);
+    }
+
+    public static ClusterService createClusterService(
```

Comment:
Can we use this everywhere in tests instead of calling constructor?

Reply:
Addressed in the revision. Refactored the UTs to use ClusterServiceUtils. This will help in avoiding changes to all such UTs in future if a new parameter say TraceRegistry is introduced in ClusterService's constructor.